db/orders.js

```diff
@@ -58,37 +56,37 @@ try{
             return;
         }
 
-        const { rows: [ order_item ] } = await client.query(`
-            UPDATE order_items
+        const { rows: [ order ] } = await client.query(`
+            UPDATE orders
             SET ${ setString }
             WHERE id=${id}
             RETURNING *;
         `, Object.values(fields));
 
-        return order_item;
+        return order;
     } catch (error) {
         console.error(error)
     }
 }
 
-async function deleteOrderItem(id) {
+async function deleteOrder(id) {
```

Comment:
I can't think of a reason we would want to delete an order at the moment but it's possible we might so we can just leave this here for now and remove it if we don't end up using it. (Just wanted to make a note in case we forget later)